packages/peregrine/lib/talons/MagentoRoute/useMagentoRoute.js

```diff
@@ -63,7 +69,7 @@ export const useMagentoRoute = props => {
             return;
         }
 
-        if (shouldFetch(routeData, store)) {
+        if (store && shouldFetch(routeData, store)) {
```

Comment:
Can move this `store` truthiness check into the `shouldFetch` utility.

Reply:
Done